cfgov/v1/models/browse_filterable_page.py

```diff
@@ -57,7 +57,18 @@ def get_form_class(self):
 
     def get_page_set(self, form, hostname):
         return filterable_context.get_page_set(self, form, hostname)
-     
+
+    def has_active_filters(self, request):
+        active_filters = False;
+        form_class = filterable_context.get_form_class()
+        filters = filterable_context.get_form_specific_filter_data(self, form_class, request.GET)
+        if bool(filters):
+            for value in filters[0].itervalues():
+                if bool(value):
```

Comment:
python should evaluate empty and non-empty values logically so the bool cast isn't necessary. Otherwise :+1: